### ISSUE TITLE
Upgrading to v2 results in error: Couldn't start client Language Server

### ISSUE THREAD

**Issue by prfmansell** [hashicorp/vscode-terraform#344]
Extension Version: 2.0.0-rc1 & rc2
OS: Windows 10
VSCode version: 1.45.1

Having installed v2 from VSIX, the following error appears on launch:

```
Launching language server: c:\Users\xxxxxx\.vscode\extensions\hashicorp.terraform-2.0.0-rc.2/lsp/terraform-ls serve
[Error - 11:22:23] Starting client failed
Error: spawn EBUSY
	at ChildProcess.spawn (internal/child_process.js:394:11)
	at Object.spawn (child_process.js:553:9)
	at c:\Users\xxxxxx\.vscode\extensions\hashicorp.terraform-2.0.0-rc.2\node_modules\vscode-languageclient\lib\main.js:354:40
```

This may also be causing syntax highlighting in terraform.tfvars files to fail:
![tfvars](https://user-images.githubusercontent.com/37519143/84021223-174a3700-a97c-11ea-915e-6576ea76ce0c.PNG)

Thanks for support

**Comment by paultyng** [comment]
The tfvars syntax highlighting is already being addressed in #349 

Not sure what the issue is with launching the LS there. Can you launch the LS manually from its installed location?

**Comment by prfmansell** [comment]
LS launching issue seems to have stopped, but now seeing: 
`Unable to find terraform.exe: exec: "terraform.exe": executable file not found in %PATH%`
I can see this is already reported in #370 so happy to close this issue in favour of that.
